python/py_helpers.py

```diff
@@ -66,24 +66,42 @@ def find_function(self, func):
                     return Node(node)
         return Node()
 
+    def find_async_function(self, func):
+        if not self._has_body():
+            return Node()
+        for node in self.tree.body:
+            if isinstance(node, ast.AsyncFunctionDef):
+                if node.name == func:
+                    return Node(node)
+        return Node()
```

Comment:
The only thing I'm not sure about is the necessity of having a separate `find` for async functions. Can we just use `find_function` for both? Or do you want to be able to check that a function exists and is async?

Reply:
I'd like to be able to check that the function is async. By the way, considering that we paused the [prime number generator project](https://github.com/freeCodeCamp/freeCodeCamp/pull/54246), we don't even need it soon.

Reply:
Cool, that makes sense. We can add this now, even if we won't need it immediately.